routes/studios.js

```diff
@@ -1,16 +1,17 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const csurfProtection = require('csurf')({ cookie: true });
+const router = express.Router();
 
 const {
   getStudios, postStudio, updateStudio, deleteStudio
 } = require('../controllers/studios.controller');
 
-router.get('/studios', getStudios, (req, res) => {
-  res.render('studios', { title: 'Studios', studios: res.locals.studios });
+router.get('/studios', csurfProtection, getStudios, (req, res) => {
+  res.render('studios', { title: 'Studios', studios: res.locals.studios, csurfToken: req.csrfToken() });
```

Comment:
Our inline template data is growing in size, what are your thoughts on factoring this out into it's own variable? Pros/cons?

Reply:
```suggestion
  const studioTemplateData = {
    title: 'Studios', 
    studios: res.locals.studios,
    csurfToken: req.csrfToken()
  };
  
  res.render('studios', studioTemplateData);
```
You mean something like this? 
I think the pros of doing this is better readability which allows developers to understand what is going on.

Reply:
Yeah, that looks nice!